package.json

```diff
@@ -64,13 +66,11 @@
     "eslint-plugin-prefer-arrow": "^1.2.1",
     "jest": "^24.5.0",
     "rimraf": "^2.6.2",
-    "single-spa": "^5.1.1",
+    "rollup": "^2.10.7",
+    "rollup-plugin-typescript2": "^0.27.1",
+    "single-spa": "^5.5.1",
     "ts-jest": "^24.0.0",
     "typescript": "3.2.4",
     "webpack": "^4.29.6"
-  },
-  "dependencies": {
-    "semver": "^6.2.0",
-    "single-spa": ">= 5.1.1"
```

Comment:
Why was this dependency moved to devDependencies?

Reply:
I will move back to deps from devDeps.